vspace/src/main/java/edu/asu/diging/vspace/core/aspects/ExhibitionDataAspect.java

```diff
@@ -118,7 +128,7 @@ private Object redirectRequest(ProceedingJoinPoint jp, String spaceId, String mo
             return "maintenance";
         }
         // If the space and module Id is not found, show message on screen.
-        if(space==null && module==null) {
+        if(space==null && module==null && !methodName.equals(ABOUT_PAGE_PUBLIC_METHOD_NAME)) {
```

Comment:
This will lead to bugs down the road. What if the controller method gets renamed, I guarantee this will be forgotten and it won't work anymore. Just don't put the controller for the about page in the exhibit package, so that the aspect never gets called. Or does it need anythings that's done in the aspect?

Reply:
What about this comment?